quarkus/service/src/main/resources/application-it.properties

```diff
@@ -0,0 +1,47 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Configuration specific to integration tests
+# Note: this file MUST be placed in the src/main/resources directory, not src/intTest/resources!
+
+quarkus.http.limits.max-body-size=1000000
+quarkus.http.port=0
```

Comment:
For posterity, it seems that with integration tests, it's this property that needs to be set to zero, and not `quarkus.http.test-port`.